temporal-sdk/src/main/java/io/temporal/internal/statemachines/WorkflowTaskStateMachine.java

```diff
@@ -52,7 +53,9 @@ void workflowTaskStarted(
   private final long workflowTaskStartedEventId;
   private final Listener listener;
 
-  private long currentTimeMillis;
+  private long eventTimeOfTheLastWorkflowStartTask;
+  // TODO write a comment describing the difference between workflowTaskStartedEventId and
+  // startedEventId
```

Comment:
Another TODO probably worth resolving in this PR

Reply:
I don't want to tackle it right now. I don't know the difference and I don't want to dig into this right now. That why I just throw a TODO for a better time.